zio-config/src/main/scala/zio/config/Config.scala

```diff
@@ -37,4 +41,21 @@ object Config {
 
   final case class Or[A, B](left: Config[A], right: Config[B]) extends Config[Either[A, B]]
 
+  def int(path: String): Config[Int] = Config.Source(path, PropertyType.IntType)
+
+  def double(path: String): Config[Double] = Config.Source(path, PropertyType.DoubleType)
+
+  def string(path: String): Config[String] = Config.Source(path, PropertyType.StringType)
+
+  def long(path: String): Config[Long] = Config.Source(path, PropertyType.LongType)
+
+  def short(path: String): Config[Short] = Config.Source(path, PropertyType.ShortType)
+
+  def uri(path: String): Config[URI] = Config.Source(path, PropertyType.UriType)
+
+  def read[A](config: => Config[A]): Read[A] = Read.read[A](config)
+
+  def write[A](config: => Config[A]): Write[A] = Write.write[A](config)
+
+  def report[A](config: => Config[A]): Report[A] = Report.report[A](config)
```

Comment:
I would imagine the `read`, `write` and `report` to be still part of the package object and rest of the functions here.

The main import for the user could be , for example,
`import zio.config._, Config._`

Reply:
sure, `read`, `write` and `report` moved back.